doc/src/Run_options.rst

```diff
@@ -503,6 +504,37 @@ e.g. the *nfile* and *fileper* keywords.  See the
 
 ----------
 
+.. _restart2info:
+
+**-restart2info restartfile keyword ...**
+
+.. versionadded:: TBD
+
+Write out some info about the restart file and and immediately exit.
+This is the same operation as if the following 2-line input script were
+run:
+
+.. code-block:: LAMMPS
+
+   read_restart restartfile
+   info system group compute fix
+
+The specified restartfile name may contain the wild-card character "\*".
+The restartfile name may also contain the wild-card character "%".  The
+meaning of these characters is explained on the :doc:`read_restart
+<read_restart>` documentation.  The use of "%" means that a parallel
+restart file can be read.  Note that a filename such as file.\* may need
+to be enclosed in quotes or the "\*" character prefixed with a backslash
+("\") to avoid shell expansion of the "\*" character.
+
+Optional keywords may follow the restartfile.  Those would have to be
```

Comment:
These must be valid keywords ...

Also on the info doc page, the args are "computes" and "fixes" (plural) which is not what it says here or up above in the code-block example.

Also, the info doc page section on Syntax should clarify that all the keywords except "out" and "styles" do not take any values.  It was a bit confusing to me to read.  I think it should read like most (if not all) of the other doc pages, e.g. the "keywords" section of the velocity command:

args = one or more keywords may be appended
keyword = out, all, system, ...
.. parsed-literal ...
only the out and styles keywords require values, the others are just keywords
out value = screen, log, append filename ...
styles value = all, angle, ...

Reply:
I just realized the info styles keyword can have multiple values, so that should be clarified in the Syntax section as well.  At least one of the Examples should illustrate that.

Reply:
Also, this -restart2info section should clarify if everything after the initial "restartfile" arg can be legitimate syntax for the "info" command.  That is more than just a list of keywords, b/c the info command allows more than that ?  Does the code simply terminate that info syntax with a command-line arg with a leading dash?

Reply:
> Does the code simply terminate that info syntax with a command-line arg with a leading dash?

The argument passing code was just copied from -restart2data and -restart2dump, so it works the same way. I don't see any remarks about leading dashes in their docs.

Reply:
> everything after the initial "restartfile" arg can be legitimate syntax for the "info" command. That is more than just a list of keywords, b/c the info command allows more than that ?

Docs currently say: "Optional keywords may follow the restartfile argument.  Those would
have to be valid keywords for the :doc:`info command <info>`." that says, *any* valid info command argument can be used. This is the same as for the other -restart2xxx only than some keywords are already added to info. This is not a big deal since the same keyword may be used multiple times without bad effects and you can use "all" without causing an error.

If we need changes beyond that, we will need them for the other -restart2XXX commands as well.

Reply:
@sjplimp Please note that the info command was originally conceived for interactive use in lammps-shell, so it does argument parsing in a way that is convenient for interactive use and therefore differs from other parts of LAMMPS:
- keywords may be abbreviated up to the characters that makes them unique (so fix and fixes is valid)
- unknown keywords or keywords with unknown arguments are ignored
- keywords that select different info sections are cumulative. You can turn sections on, but not off. "all" just turns all on and then keywords like "fixes" or "computes" are already active

I am surprised that this is causing such a big fuss now after it has been around for quite some time.

Reply:
For your first 2 bullets, the info doc page does not explain that (it should).
For the -restart2info doc page, you shouldn't use "fix compute" in one place and "fixes computes" in another.
The Syntax section of the info doc page should be formatted like every other LAMMPS command doc page - that is all I am saying.  You can make the changes in this PR or another PR or I can do it.